setup.py

```diff
@@ -16,17 +16,13 @@
     "pytest-django==4.5.0",
     "pytest-cov==3.0.0",
     "pytest-pythonpath==0.7.3",
-    "psycopg2>=2.3.1",
     "coverage==6.0",
```

Comment:
ditto, for another PR

Reply:
Opened https://github.com/wagtail/wagtail-factories/issues/86